src/Jackett.Common/Indexers/PassThePopcorn.cs

```diff
@@ -253,5 +258,18 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
 
             return releases;
         }
+
+        public override async Task<byte[]> Download(Uri link)
+        {
+            var authHeaders = new Dictionary<string, string>()
+            {
+                { "ApiUser", configData.User.Value },
+                { "ApiKey", configData.Key.Value }
+            };
+
+            var response = await base.RequestBytesWithCookies(link.ToString(), headers: authHeaders);
```

Comment:
If the headers are not required, remove this code. If the previous code works for you, the issue is in @RoloSoze configuration and this PR is not required at all.

Reply:
This part is probably unnecessary. I plan to undo these changes.